test/testUtils/FormTester.scala

```diff
@@ -1,15 +1,16 @@
 package testUtils
 
 import org.scalatest.matchers.must.Matchers._
+import testUtils.DefaultMockFormOptions.MockInputOption
 
 class FormTester(defaultOptions: List[MockInputOption], smallCheckbox: String = " govuk-checkboxes--small") {
-  def generateWaysToIncorrectlySubmitAForm(): Seq[Seq[(String, String)]] = {
+  def generateWaysToIncorrectlySubmitAForm(value: String = "true", combineOptionNameWithValue: Boolean = false): Seq[Seq[(String, String)]] = {
     val possibleOptions: Seq[String] = defaultOptions.map(_.name)
     val optionsToSelectToGenerateFormErrors =
       for {
         numberRangeOfOptionsToSelect <- (1 until possibleOptions.length).toList
         optionsToSelect <- possibleOptions.combinations(numberRangeOfOptionsToSelect)
-      } yield optionsToSelect.map(option => (option, "true"))
+      } yield optionsToSelect.map(option => (option, if (combineOptionNameWithValue) s"$option $value" else value))
 
     optionsToSelectToGenerateFormErrors
```

Comment:
I know you didn't change this but you don't need the variable here, you can just have the for comprehension as the last line.

Reply:
Ah, yes. I think I wrote this originally. The reason why was just to give the reader a little understanding as to what it was returning but I will change the method name to something like `generateOptionsToSelectToGenerateFormErrors` so that it's no longer necessary. 👍